pulsar/resource_pulsar_namespace.go

```diff
@@ -230,6 +230,30 @@ func resourcePulsarNamespace() *schema.Resource {
 					},
 				},
 			},
+			"topic_auto_creation": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enable": {
+							Type:     schema.TypeBool,
+							Required: true,
+						},
+						"type": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: validatePartitionedTopicType,
+							Default:      "non-partitioned",
+						},
+						"partitions": {
+							Type:     schema.TypeInt,
+							Optional: true,
```

Comment:
Just confirm: the default value is `0`, right?

Reply:
yes